src/common/webrequest_curl.cpp

```diff
@@ -524,6 +524,13 @@ void wxWebRequestCURL::Start()
     StartRequest();
 }
 
+void wxWebRequestCURL::SetTimeouts(long connectionTimeoutMs,
+                                   long dataTimeoutMs)
+{
+    long timeout = connectionTimeoutMs + dataTimeoutMs;
+    wxCURLSetOpt(m_handle, CURLOPT_TIMEOUT_MS, timeout);
```

Comment:
We need to set `CURLOPT_CONNECTTIMEOUT_MS` too, the documentation even mentions it, but the code doesn't do it for some reason?